neural_compressor/onnxrt/quantization/quantize.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, Tuple
+
+import onnx
+
+from neural_compressor.common.base_config import BaseConfig, ComposableConfig, config_registry
+from neural_compressor.common.logger import Logger
+from neural_compressor.onnxrt.quantization.config import FRAMEWORK_NAME
+from neural_compressor.onnxrt.utils.utility import algos_mapping
+
+logger = Logger().get_logger()
+
+
+def need_apply(configs_mapping: Dict[Tuple[str, callable], BaseConfig], algo_name):
+    return any(config.name == algo_name for config in configs_mapping.values() if hasattr(config, "name"))

```

Comment:
if we expand config in algo_entry, here should be quant_config.name == algo_name